### DISCUSSION TITLE
Power Button Bug With Pi Zero-W

### DISCUSSION THREAD

**Discussion by Bautar** [rern/rAudio#1650] [Bug Reports]
I’m running a Raspberry Pi Zero-W with a Pimoroni Phat DAC, Rotary Encoder, and Power Button connected to pin 5 for both on and off functions.  
The button stopped working weeks if not months ago but I just now looked into it & found the “On” dropdown arrow where I would confirm/select the pin the button is wired to is missing. The word is there but the white arrow is missing & I cannot access the dropdown menu. The arrow and menus are available and working for Off and LED. 
suspect button is not working due to this?
Image attached. 

Thanks,

bautar

<img width="414" alt="image" src="https://github.com/rern/rAudio/assets/99979632/ba242ad5-bc25-40dc-912c-31f8f0ccb044">

**Comment by rern** [comment]
Please try the [latest image](https://github.com/rern/rAudio/releases/tag/i20240331).
